src/ImageSharp/Formats/Jpeg/JpegDecoder.cs

```diff
@@ -38,5 +38,7 @@ public IImageInfo Identify(Configuration configuration, Stream stream)
                 return decoder.Identify(stream);
             }
         }
+        
+        public Image Decode(Configuration configuration, Stream stream) => this.Decode<Rgba32>(configuration, stream);
```

Comment:
This will become `Rgb24` yes?

Reply:
It will delegate into a method of `JpegDecoderCore` which can then decide what pixel type to use in the resulting `Image<T>`. For grayscale jpegs we can return `Image<Gray8>` (if worth).

Designing the code sharing between the specialized (generic) and "native pixel type" (non-generic) methods is an interesting question. Probably even more challenging for `PngDecoder`.

Reply:
That makes sense. Thinking about it shouldn’t be too bad for png either.